odbc/src/api/bind_col_tests.rs

```diff
@@ -466,4 +403,832 @@ mod unit {
             let _ = Box::from_raw(buffer as *mut WChar);
         }
     }
+
+    #[test]
+    fn test_binding_arrays_when_rowset_size_evenly_divides_resultset_size() {
+        // Set up MongoHandle
```

Comment:
So far I've only added tests that successfully fetch data and bind columns. I didn't add any tests for error handling yet because there's a lot of possible errors that could come up so I wasn't sure what the best way to test that would be. For example, the cases where one row causes an error, every row is an error, a row can be fetched but then binding fails, etc.

Reply:
Ticket: https://jira.mongodb.org/browse/SQL-2010